charts/generic/k8-custom-metrics-apiserver/Readme.md

```diff
@@ -0,0 +1,10 @@
+# Remote namespace 
```

Comment:
@dharmaanu , this is a custom solution and don't have kubernetes resource associate, we don't need this chaos-chart.